client/src/components/subscription/subscriptionForm.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+
+/** TO DO
```

Comment:
```suggestion
/** TODO:
```

It's common to type it out this way for `TODO:` and `FIXME:` so that they are highlighted in the editor if you have the extension. Example:

<img width="1156" alt="Screen Shot 2022-01-12 at 11 23 58 PM" src="https://user-images.githubusercontent.com/25065329/149265699-dca7d811-7ed1-47d4-b0d7-5c6ff96b960a.png">

Reply:
Gotcha, will do this going forward.